Runtime/Scripts/RGBotConfigs/DefaultActions/DrawIndicator.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RegressionGames.RGBotConfigs.DefaultActions
+{
+    
+    /**
+     * This action uses Gizmos to draw an indicator at a specific position in a scene.
+     * Takes the following parameters:
+     *  If modifying an indicator (either new or old)
+     *    - name (string): And identifier for the indicator
+     *    - position (Vector3): The position to place the indicator
+     *    - color (Color): The color of the indicator
+     *    - size (float): The size of the indicator
+     *  If removing the indicator:
+     *    - name (string): The indicator to remove
+     *    - remove (boolean): Set to true
+     *  If removing all indicators:
+     *    - removeAll (bool): Set to true
+     */
+    public class DrawIndicator: RGAction
+    {
+
+        private Dictionary<string, Tuple<Vector3, Color, float>> _indicators = new ();
+
+        public override string GetActionName()
+        {
+            return "DrawIndicator";
+        }
+
+        public override void StartAction(Dictionary<string, object> input)
+        {
+
+            if ((bool) input.GetValueOrDefault("removeAll", false))
+            {
+                _indicators.Clear();
+                return;
+            }
+
+            var indicatorName = (string) input["name"];
+            if ((bool) input.GetValueOrDefault("remove", false))
+            {
+                _indicators.Remove(indicatorName);
+            }
+            else
+            {
+                var targetPosition = (Vector3) input["position"];
+                var color = (Color) input["color"];
+                var size = (float) input["size"];
+                var indicatorParams = Tuple.Create(targetPosition, color, size);
+                _indicators[indicatorName] = indicatorParams;
+            }
```

Comment:
You can see hear that my actions are "overloaded". Since my actions keep the references to the indicators (and also line and text in the other actions), I decided to back the remove operation into this as well. I could have put all indicator/line/text logic into somewhere on RGEntity, but that seemed much worse.